File: EPRO.Library.v3.5/EPRO.Library.v3.5/Caching/Exceptions/NotCachedException.cs
using CODERiT.Logger.v._3._5.Exceptions;

namespace ELibrary.Standard.Caching.Exceptions
{
    public class NotCachedException : EException
    {
        public NotCachedException(string FormTypeName) : base(string.Format("Invalid Code Usage. The requested Form [ {0} ] is NOT cached!!!", FormTypeName))
        {
        }
    }
}

File: EPRO.Library.v3.5/EPRO.Library.v3.5/Caching/Exceptions/NotCachedException.vb
Imports CODERiT.Logger.v._3._5.Exceptions

Namespace Caching.Exceptions
    Public Class NotCachedException
        Inherits EException

        Sub New(ByVal FormTypeName As String)
            MyBase.New(
                String.Format(
                    "Invalid Code Usage. The requested Form [ {0} ] is NOT cached!!!",
                    FormTypeName
                    )
                )
        End Sub

    End Class

End Namespace